HelloTest.java

```diff
@@ -0,0 +1,23 @@
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+import static org.junit.Assert.assertEquals;
+import org.junit.Test;
+import org.junit.Before;
+import org.junit.After;
+
+public class HelloTest {
+	private final ByteArrayOutputStream outContent = new ByteArrayOutputStream();
+	@Before	
+	public void setUpStreams() {
+   		System.setOut(new PrintStream(outContent));	
```

Comment:
Before setting to something new, save current System.out value to your field (see above comment).